shell/src/main/java/alluxio/cli/fs/command/CpCommand.java

```diff
@@ -69,6 +69,9 @@
   private static final Logger LOG = LoggerFactory.getLogger(CpCommand.class);
   private static final String COPY_SUCCEED_MESSAGE = "Copied %s to %s";
   private static final String COPY_FAIL_MESSAGE = "Failed to copy %s to %s";
+  private static final int OPTION_NOT_SET = -1;
+  private static final int COPY_FROM_LOCAL_BUFFER_SIZE_DEFAULT = 8 * Constants.MB;
```

Comment:
why are these defaults different?

Reply:
I used the old values, when copying from Alluxio to local, we may want to use a larger buffer because Alluxio block size is larger.